tests/test_user/test_query_user.py

```diff
@@ -84,3 +85,13 @@ def test_get_users_by_location_and_role(self):
         CommonTestCases.user_token_assert_in(
             self, get_users_by_location_and_role, "No users found"
             )
+
+    def test_filter_users_by_location_no_users(self):
```

Comment:
Make this test function name more descriptive. The name is different from what is being tested

Reply:
Thanks @Bonifase, all the above have been named appropriately now.